common/selectors/selectAndDedupe.js

```diff
@@ -13,14 +13,12 @@ const dedupe = require("lib/dedupe");
 module.exports = function selectAndDedupe(group) {
   return group.reduce((result, options, index, arr) => {
     let current;
+    let sites = options.defaults ? options.sites.concat(options.defaults) : options.sites;
```

Comment:
I assume `options.defaults` is null or an empty array if `optons.sites` has enough links, correct? Otherwise, the defaults might exclude the actual topsites in some cases.